epub33/core/index.html

```diff
@@ -1411,6 +1411,11 @@ <h4>Resource locations</h4>
 				<p>Storing all resources inside the EPUB container is strongly encouraged whenever possible as it allows
 					users access to the entire presentation regardless of connectivity status.</p>
 
+				<p>When resources have to be located outside the EPUB container, EPUB creators are strongly RECOMMENDED
+					to securely reference them via the <code>https</code> URI scheme [rfc7230] to limit the threat of
+					exposing their publications, and users, to network attacks. Reading systems may not load remote
```

Comment:
Just a minor comment: not all resources on the Web are available through https, and this is not under the control of the author. Maybe it is better to say something like:

>...to securely reference them, whenever possible, via the ...

Reply:
Isn't that baked into it being a recommendation instead of a requirement?  If we add "whenever possible" we reduce it to an untestable assertion, so effectively an informative "should".

Remote resources are things the author is explicitly bringing into the publication, not things they're linking out to, so in this case I think the burden should be on the author to only reference secure resources given it's exposing users and RSes to an attack vector.

Reply:
> Isn't that baked into it being a recommendation instead of a requirement? I

Oops, you are right. Forget this.